km_api/know_me/urls.py

```diff
@@ -10,11 +10,13 @@
     url(r'^emergency-items/(?P<pk>[0-9]+)/$', views.EmergencyItemDetailView.as_view(), name='emergency-item-detail'),   # noqa
     url(r'^media-resources/(?P<pk>[0-9]+)/$', views.MediaResourceDetailView.as_view(), name='media-resource-detail'),   # noqa
     url(r'^items/(?P<pk>[0-9]+)/$', views.ProfileItemDetailView.as_view(), name='profile-item-detail'),                 # noqa
+    url(r'^items/(?P<pk>[0-9]+)/list-entries/$', views.ListEntryListView.as_view(), name='list-entry-list'),          # noqa
+    url(r'^list-entries/?(P<pk>[0-9]+)/$', views.ListEntryDetailView.as_view(), name='list-entry-detail'),              # noqa
     url(r'^profiles/(?P<pk>[0-9]+)/$', views.ProfileDetailView.as_view(), name='profile-detail'),                       # noqa
     url(r'^profiles/(?P<pk>[0-9]+)/topics/$', views.ProfileTopicListView.as_view(), name='profile-topic-list'),         # noqa
     url(r'^topics/(?P<pk>[0-9]+)/$', views.ProfileTopicDetailView.as_view(), name='profile-topic-detail'),              # noqa
     url(r'^topics/(?P<pk>[0-9]+)/items/$' ,views.ProfileItemListView.as_view(), name='profile-item-list'),              # noqa
-    url(r'^users/$', views.KMUserListView.as_view(), name='km-user-list'),
+    url(r'^users/$', views.KMUserListView.as_view(), name='km-user-list'),                                              # noqa
```

Comment:
This line doesn't need the `# noqa` comment because it doesn't exceed the maximum line length.